AllASPFuncs.py

```diff
@@ -678,6 +686,18 @@ def open_parq(ID, path = '/cwork/mat90/RomanDESC_sims_2024/roman_rubin_cats_v1.1
     df = pd.read_parquet(path+'/snana_'+str(ID)+'.parquet', engine='fastparquet')
     return df
 
+
+def star_to_loc(starID, parq, band, path):
```

Comment:
I'd prefer a name that tells me I'm going to get an (RA, Dec).   `loc` could mean many things; in particular I would have assume it might be some location in a table / index.

Reply:
But my other question is why does this have to be a separate function from the one for supernova?